qrenderdoc/Windows/Dialogs/SettingsDialog.ui

```diff
@@ -287,6 +287,23 @@ e.g. 1000 * 10 = 1e4</string>
            </widget>
           </item>
           <item row="10" column="0">
+           <widget class="QLabel" name="label_4">
+            <property name="toolTip">
+             <string>Formatting mode to use for fields marked as a byte offset or a byte size</string>
+            </property>
+            <property name="text">
+             <string>Offset or size fields format mode</string>
```

Comment:
I couldn't decide if this should be
"Offset or size"
or
"Offset or Size"

else where have been referring to this field type as Offset or Size

Reply:
I think either is probably fine, I slightly prefer this way.